dropwizard-jdbi/src/main/java/io/dropwizard/jdbi/DBIFactory.java

```diff
@@ -28,18 +31,36 @@
     private static class SanerNamingStrategy extends DelegatingStatementNameStrategy {
         private SanerNamingStrategy() {
             super(NameStrategies.CHECK_EMPTY,
-                  NameStrategies.CONTEXT_CLASS,
-                  NameStrategies.CONTEXT_NAME,
-                  NameStrategies.SQL_OBJECT,
-                  new StatementNameStrategy() {
-                      @Override
-                      public String getStatementName(StatementContext statementContext) {
-                          return RAW_SQL;
-                      }
-                  });
+                    NameStrategies.CONTEXT_CLASS,
```

Comment:
You are right. I just formatted the class in my IDE instinctively. 
It was a bit of sloppy and I will try to avoid such changes in future.